plugins/scaffolder-backend/src/scaffolder/storage/index.ts

```diff
@@ -1,4 +1,4 @@
-import { DiskStorage } from './disk';
+import { Logger } from "winston";
```

Comment:
I think you need to enable prettier, there are a couple places where it deviates.

Reply:
For some reason it doesn't automatically format on save in vscode for me, and I thought it used to be a githook? Will have a play.